app/Resources/assets/scss/app.scss

```diff
@@ -836,3 +836,10 @@ ul.alternatives {
     cursor:pointer;
   }
 }
+
+.executive-board-table {
+    border-width: 1px;
+    border-color: #FFD700;
```

Comment:
Hva tenker du om å bruke vektor-blåfargen her for å holde oss til temaet på resten av nettsiden?